sympy/functions/elementary/complexes.py

```diff
@@ -896,18 +1064,32 @@ class periodic_argument(Function):
     logarithm. That is, given a period P, always return a value in
     (-P/2, P/2], by using exp(P*I) == 1.
 
-    >>> from sympy import exp, exp_polar, periodic_argument, unbranched_argument
+    Examples
+    ========
+
+    >>> from sympy import exp_polar, periodic_argument
     >>> from sympy import I, pi
-    >>> unbranched_argument(exp(5*I*pi))
-    pi
-    >>> unbranched_argument(exp_polar(5*I*pi))
-    5*pi
-    >>> periodic_argument(exp_polar(5*I*pi), 2*pi)
-    pi
-    >>> periodic_argument(exp_polar(5*I*pi), 3*pi)
-    -pi
-    >>> periodic_argument(exp_polar(5*I*pi), pi)
+    >>> periodic_argument(exp_polar(10*I*pi), 2*pi)
     0
+    >>> periodic_argument(exp_polar(5*I*pi), 4*pi)
+    pi
+
+    Parameters
+    ==========
+
+    ar : Expr
+        A polar number.
+
+    period : Expr
+
+        >>> from sympy import exp_polar, periodic_argument
```

Comment:
There shouldn't be examples in the parameters section.

Reply:
I guess this is based on what @sylee957 wrote. But I think it's better to explain here in plain English what the parameter does. The examples section can show what it does in examples. I think it's already explained in the main part of the docstring, so here I would just write something like "the period $P$", which references the mathematical description at the beginning of the docstring.

Reply:
Done.